db/org_members_groups.go

```diff
@@ -0,0 +1,273 @@
+package db
+
+import (
+	"context"
+	"fmt"
+	"math"
+	"time"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"go.vocdoni.io/dvote/log"
+)
+
+// OrgMembersGroup returns an organization members group
+func (ms *MongoStorage) OrganizationMemberGroup(groupID, orgAddress string) (*OrganizationMemberGroup, error) {
+	objID, err := primitive.ObjectIDFromHex(groupID)
+	if err != nil {
+		return nil, ErrInvalidData
+	}
+
+	filter := bson.M{"_id": objID, "orgAddress": orgAddress}
+	// create a context with a timeout
+	ctx, cancel := context.WithTimeout(context.Background(), defaultTimeout)
+	defer cancel()
+	// find the organization in the database
+	result := ms.orgMemberGroups.FindOne(ctx, filter)
+	var group *OrganizationMemberGroup
+	if err := result.Decode(&group); err != nil {
+		// if the organization doesn't exist return a specific error
+		if err == mongo.ErrNoDocuments {
+			return nil, ErrNotFound
+		}
+		return nil, err
+	}
+	return group, nil
+}
+
+// OrgMembersGroupsByOrg returns the list of an organization's members groups
+func (ms *MongoStorage) OrganizationMemberGroups(orgAddress string, page, pageSize int) (int, []*OrganizationMemberGroup, error) {
+	// create a context with a timeout
+	ctx, cancel := context.WithTimeout(context.Background(), defaultTimeout)
+	defer cancel()
+
+	filter := bson.M{"orgAddress": orgAddress}
+
+	// Count total documents
+	totalCount, err := ms.orgMemberGroups.CountDocuments(ctx, filter)
+	if err != nil {
+		return 0, nil, err
+	}
+	totalPages := int(math.Ceil(float64(totalCount) / float64(pageSize)))
+
+	// Calculate skip value based on page and pageSize
+	skip := (page - 1) * pageSize
+
+	// Set up options for pagination
+	findOptions := options.Find().
+		SetSkip(int64(skip)).
+		SetLimit(int64(pageSize))
+
+	// find the organization in the database
+	cursor, err := ms.orgMemberGroups.Find(ctx, filter, findOptions)
+	if err != nil {
+		return 0, nil, err
+	}
+	defer func() {
+		if err := cursor.Close(ctx); err != nil {
+			log.Warnw("error closing cursor", "error", err)
+		}
+	}()
+
+	var groups []*OrganizationMemberGroup
+	for cursor.Next(ctx) {
+		var group OrganizationMemberGroup
+		if err := cursor.Decode(&group); err != nil {
+			return 0, nil, err
+		}
+		groups = append(groups, &group)
+	}
+
+	if err := cursor.Err(); err != nil {
+		return 0, nil, err
+	}
+
+	return totalPages, groups, nil
+}
+
+// CreateOrganizationGroup Creates an organization member group
+func (ms *MongoStorage) CreateOrganizationMembersGroup(group *OrganizationMemberGroup) (string, error) {
```

Comment:
looking at the other method names, this should be CreateOrganizationMemberGroup for consistency

Reply:
or, well, change the other methods to refer to "OrganizationMembersGroup"